driver/src/contracts/gnosis_safe_gas_station.rs

```diff
@@ -0,0 +1,82 @@
+use ethcontract::U256;
```

Comment:
This should probably not be in the contracts module as it is not really doing any smart contract interaction. I think it even deserves its own module.